lib/modules/shipments_api_retriever.rb

```diff
@@ -8,6 +8,20 @@ def self.api_call(compliance_type = nil, page = 1)
     JSON.parse(response.body)
   end
 
+  def self.grouped_call(*grouping)
+    record = {}
+    response = HTTParty.get(Rails.application.secrets['species_api_url'] + '/grouped',
+                            headers: header,
+                            query: { group_by: grouping })
+    data = JSON.parse(response.body)['shipments']
+    return data if grouping.include?('issue_type') || grouping.empty?
```

Comment:
I think it would be best to do the formatting of the data from the Species+ side of things for a couple of reasons (this also includes the lines below): 

1. The Compliance Tool should be as agnostic as possible in terms of data manipulation
2. If we filter the data from Species+, we are passing fewer records via the communication channel, possibly making the overall data transfer quicker.